src/asInput/index.jsx

```diff
@@ -37,7 +37,7 @@ export const inputProps = {
 export const defaultProps = {
   onChange: () => {},
   onBlur: () => {},
-  id: newId('asInput'),
+  id: undefined,
```

Comment:
Mostly curious about why this change was made.

Reply:
The problem with this is that `defaultProps` is only defined once per import of this module. So every `asInput` created without an id specified would get this same id. The code in the constructor, which is called on every new `asInput` component, should call `newId` if this is `undefined` now.

Reply:
I was thinking about this recently - I think we should switch over to something like https://github.com/dylang/shortid ?